src/compiler/diagnosticMessages.json

```diff
@@ -6346,5 +6346,9 @@
     "Invalid value for 'jsxFragmentFactory'. '{0}' is not a valid identifier or qualified-name.": {
         "category": "Error",
         "code": 18035
+    },
+    "Non-abstract class '{0}' does not implement all abstract members of '{1}'": {

```

Comment:
Maybe the message should be "Non-abstract class '{0}' does not implement all inherited abstract members of '{1}'", to match the old ones